<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>862</width>
                <height>790</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_2" rowstretch="3,0,3" columnstretch="2,0,1">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item row="0" column="0">
                        <widget class="QTreeView" name="treeview"/>
                    </item>
                    <item row="0" column="1" colspan="2">
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QStackedWidget" name="stacked_widget">
                                    <property name="currentIndex">
                                        <number>-1</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item row="2" column="0" colspan="3">
                        <widget class="QScrollArea" name="scrollArea">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="widgetResizable">
                                <bool>true</bool>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                            <widget class="QWidget" name="pipeline_container">
                                <property name="geometry">
                                    <rect>
                                        <x>0</x>
                                        <y>0</y>
                                        <width>856</width>
                                        <height>368</height>
                                    </rect>
                                </property>
                                <property name="sizePolicy">
                                    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                                        <horstretch>0</horstretch>
                                        <verstretch>0</verstretch>
                                    </sizepolicy>
                                </property>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <property name="spacing">
                                        <number>20</number>
                                    </property>
                                    <property name="sizeConstraint">
                                        <enum>QLayout::SetDefaultConstraint</enum>
                                    </property>
                                    <property name="leftMargin">
                                        <number>20</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>50</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>20</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>50</number>
                                    </property>
                                    <item>
                                        <spacer name="horizontalSpacer_3">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeType">
                                                <enum>QSizePolicy::Expanding</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="pipeline_execute_btn">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>80</width>
                                                    <height>80</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="icon">
                                                <iconset theme="media-playback-start"/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </widget>
                    </item>
                    <item row="1" column="0" colspan="3">
                        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,3,1,4">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>5</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QComboBox" name="pipeline_cb">
                                    <property name="sizeAdjustPolicy">
                                        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
                                    </property>
                                    <property name="placeholderText">
                                        <string/>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="import_pipeline_btn">
                                    <property name="text">
                                        <string>Import</string>
                                    </property>
                                    <property name="icon">
                                        <iconset theme="download">
                                            <normaloff>.</normaloff>.
                                        </iconset>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="pipeline_name_edit">
                                    <property name="maxLength">
                                        <number>25</number>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                    <property name="placeholderText">
                                        <string>Pipeline Name</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="export_pipeline_btn">
                                    <property name="text">
                                        <string>Export</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="add_processor_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimumSize">
                                        <size>
                                            <width>0</width>
                                            <height>0</height>
                                        </size>
                                    </property>
                                    <property name="text">
                                        <string/>
                                    </property>
                                    <property name="icon">
                                        <iconset theme="list-add">
                                            <normaloff>.</normaloff>.
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>24</width>
                                            <height>16</height>
                                        </size>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
